packages/flutter_html_table/lib/flutter_html_table.dart

```diff
@@ -11,15 +11,21 @@ CustomRender tableRender() =>
     CustomRender.widget(widget: (context, buildChildren) {
       return Container(
         key: context.key,
-        margin: context.style.margin?.nonNegative,
+        //TODO(Sub6Resources): This needs to be computed with Units!!
+        margin: EdgeInsets.only(
+          left: context.style.margin?.left?.value.abs() ?? 0,
```

Comment:
Nit: `.abs()` would give a margin value of, for example, 14 if you were to use `-14px` in css. While we don't support negative values, clamping to 0 is slightly better?

Reply:
Fixed by replacing container with `CssBoxWidget`